observation_portal/accounts/apps.py

```diff
@@ -3,3 +3,7 @@
 
 class AccountsConfig(AppConfig):
     name = 'observation_portal.accounts'
+
+    def ready(self):
+        import observation_portal.accounts.signals.handlers  # noqa
```

Comment:
Not familiar with this, but I am guessing it's to enable signals and handlers in the accounts app?

Reply:
yeah, its so the signal handler @receiver gets loaded in with the app